gpytorch/math/functions/lazy_kronecker_product/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .kp_interpolated_toeplitz_gp_marginal_log_likelihood import KPInterpolatedToeplitzGPMarginalLogLikelihood
```

Comment:
I'd prefer to have `KroneckerProduct` rather than `KP`, but the name is already long...